tests/input/credential-issuer-object-id-missing-fail.json

```diff
@@ -3,9 +3,12 @@
     "https://www.w3.org/ns/credentials/v2"
   ],
   "type": [
-    "VerifiableCredential"
+    "VerifiableCredential",
+    "NoIssuer"
```

Comment:
this will cause the VC to fail on issuers and verifiers using jsonld in safe mode so we should probably avoid this. otherwise I love this PR and good work.

Reply:
Yeah...I suppose this would fail for multiple reasons, so narrowing it to just the issuer.id being null would be best.

Reply:
This has been addressed, I no longer have the `NoIssuer` type